common/metrics/defs.go

```diff
@@ -2796,6 +2797,7 @@ var MetricDefs = map[ServiceIdx]map[int]metricDefinition{
 		TaskWriteThrottlePerTaskQueueCounter:      NewRollupCounterDef("task_write_throttle_count_per_tl", "task_write_throttle_count"),
 		TaskWriteLatencyPerTaskQueue:              NewRollupTimerDef("task_write_latency_per_tl", "task_write_latency"),
 		TaskLagPerTaskQueueGauge:                  NewGaugeDef("task_lag_per_tl"),
+		NoRecentPollerTasksPerTaskQueueCounter:    NewRollupCounterDef("no_poller_tasks_per_tl", "no_poller_tasks"),
```

Comment:
Rob suggested in #3143 that we use `_per_taskqueue` names for metrics (`tl` is from Cadence, which calls it a task list).  I think we should create new ones with `_per_taskqueue` and then migrate the old ones later.